tutor/src/components/task/progress/milestones.cjsx

```diff
@@ -129,13 +129,20 @@ MilestonesWrapper = React.createClass
     currentStep: currentStep
     crumbs: crumbs
 
+  componentWillMount: ->
+    bStyle = document.body.style
+    @_bodyOverflowStyle = bStyle.overflow
+    bStyle.overflow = 'hidden'
+
   componentDidMount: ->
     @switchCheckingClick()
     @switchTransitionListen()
 
   componentWillUnmount: ->
     @switchCheckingClick(false)
     @switchTransitionListen(false)
+    bStyle = document.body.style
+    bStyle.overflow = @_bodyOverflowStyle
 
```

Comment:
it's awesome that you cache the original style and restore it vs just blindly setting it to whatever..